### PR TITLE
TestGrid: Split Conformance Tabs into Subdirectory

### PR DESCRIPTION
**Pull request by chases2**
Separating the conformance tabs further is left to future PRs, as owners become more clear
xref #13541 

/cc @aojea 
/cc @BenTheElder 
/cc @spiffxp

### CODE REVIEW

File: config/testgrids/conformance/kind/conformance-kind.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+dashboards:
+- name: conformance-kind
```
Comment (BenTheElder) [inline config/testgrids/conformance/kind/conformance-kind.yaml]: should we be moving all of these?

Comment (aojea) [reply config/testgrids/conformance/kind/conformance-kind.yaml]: you mean independently in its own folder or to the conformance-all?
I don´t have a strong opinion but I think that it should belong to sig-testing/kind and conformance-all

Comment (chases2) [reply config/testgrids/conformance/kind/conformance-kind.yaml]: The conformance dashboards seem large enough that they should be split into their own file(s). 

This change doesn't change the contents of the dashboards; it wasn't on `sig-testing`, and it still isn't, but it keeps it on both conformance tabs.

@aojea, if you're looking to add this to sig-testing also, I would copy the tab in question and put it in the sig-testing file as you had before, but no need to delete anything in conformance.

**Comment by chases2** [comment]
Is the added owners files okay?
/cc @munnerz @BenTheElder @neolit123

File: config/testgrids/conformance/kind/conformance-kind.yaml (hunk 1 above)
Comment (aojea) [reply config/testgrids/conformance/kind/conformance-kind.yaml]: I think this is ok, we can follow up if necessary with a new pull request

**Comment by aojea** [comment]
Sgtm, i let others approve
Thanks Sean

**Comment by neolit123** [comment]
i'm fine to be in the OWNERS file for `config/testgrids/conformance/kind`, but who should be in the OWNERS file in `config/testgrids/conformance/` (there isn't one yet).
..or should this be a responsibility of the parent ONWERS file in `config/testgrids`?

probably @spiffxp should decided.

File: config/testgrids/conformance/kind/conformance-kind.yaml (hunk 1 above)
Comment (BenTheElder) [reply config/testgrids/conformance/kind/conformance-kind.yaml]: I'm confused that the conformance-kind dashboard here only seems to have a single tab

Comment (BenTheElder) [reply config/testgrids/conformance/kind/conformance-kind.yaml]: it should have 11 currently

**Comment by spiffxp** [comment]
I'm not sure I buy into sub-folders for each of the tabs (I leave that to whatever's easier for kind), but I guess it makes sense for `config/testgrids/conformance/OWNERS` to exist, this is one of the more likely to configs to get populated by externally provided results in arbitrary GCS buckets.

Put myself and @timothysc as approvers for `config/testgrids/conformance/OWNERS`

File: config/testgrids/conformance/kind/conformance-kind.yaml (hunk 1 above)
Comment (chases2) [reply config/testgrids/conformance/kind/conformance-kind.yaml]: @BenTheElder I believe the others tabs are prow annotations, and aren't located in either YAML doc

Comment (BenTheElder) [reply config/testgrids/conformance/kind/conformance-kind.yaml]: [trying to think why this one is the sole one not in an annotation]

Comment (chases2) [reply config/testgrids/conformance/kind/conformance-kind.yaml]: While there's a prow job named `ci-kubernetes-kind-conformance-latest-1-14`, I don't think there's one named `ci-kubernetes-kind-conformance-latest-1-14-arm64`

Comment (BenTheElder) [reply config/testgrids/conformance/kind/conformance-kind.yaml]: derp. this one is from third party CI. TBH I'm slightly thinking we should move it to another dashboard but that's orthogonal do this work.

thanks

**Pull request merged**